vcl_templates/www.vcl.erb

```diff
@@ -64,7 +64,7 @@ backend F_eks_origin {
     .probe = {
         .request =
             "HEAD <%= config['probe'] %> HTTP/1.1"
-            "Host: <%= config.fetch('origin_hostname') %>"
+            "Host: <%= config.fetch('healthcheck_hostname', config['eks_origin_hostname']) %>"
```

Comment:
We should probably not default back to the origin header, as it removes the safety of the health check.